### DISCUSSION TITLE
"this video may be inappropriate for some users"

### DISCUSSION THREAD

**Discussion by nicojlt** [Tyrrrz/YoutubeDownloader#275] [Q&A]
Hi, 

Some videos needs you to be connected to be viewed. (age related / adult content)

I would like to know if there is a way to download it ?

example :  https://www.youtube.com/watch?v=niCifL9-XsM

![image](https://user-images.githubusercontent.com/64739912/177940007-3b9e7afa-c936-4f63-81fa-90d8196e99cc.png)

thanks :)

**Comment by Tyrrrz** [comment]
Not right now, no. You can retry a few times, sometimes it eventually lets you download it.
